packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -137,18 +117,11 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
    */
   private updateMetadata(
     transaction: PersistenceTransaction,
-    changedKeys: DocumentKeySet,
     sizeDelta: number
   ): PersistencePromise<void> {
     return this.getMetadata(transaction).next(metadata => {
       metadata.byteSize += sizeDelta;
-      return this.setMetadata(transaction, metadata).next(() => {
-        if (this.keepDocumentChangeLog) {
-          return documentChangesStore(transaction).put({
-            changes: this.serializer.toDbResourcePaths(changedKeys)
-          });
-        }
-      });
```

Comment:
The method comment still says "Updates the document change log and ..."  Perhaps do a grep over the codebase for "changelog" and "change log" to make sure there aren't other stale instances?